packages/commons/src/libs/migrations.ts

```diff
@@ -555,6 +556,57 @@ export const Migrations: {
         });
       });
     }
+  },
+  {
+    id: 29,
+    migrationFunction: (environment: Environment) => {
+      environment.routes.forEach((route: Route) => {
+        route.responses.forEach((routeResponse) => {
+          let body: string | undefined;
+          let valueIndex = -1;
+          // Get effective response body from body parameter or databucket value
+          // We are not updating if bodyType is FILE.
+          if (routeResponse.bodyType === BodyTypes.INLINE) {
+            body = routeResponse.body;
+          } else if (routeResponse.bodyType === BodyTypes.DATABUCKET) {
+            valueIndex = environment.data.findIndex(
+              (dataItem) => dataItem.id === routeResponse.databucketID
+            );
+            body = valueIndex > -1 ? environment.data[valueIndex].value : body;
+          }
```

Comment:
I think you could simplify a bit and just loop over all the databuckets, linked to a response or not. 

```
environment.data.foreach(...)
```

Reply:
Changed. Will be available in next commit.